build/yamls/base/conf/antrea-agent.conf

```diff
@@ -32,3 +32,10 @@
 # CIDR Range for services in cluster. It's required to support egress network policy, should
 # be set to the same value as the one specified by --service-cluster-ip-range for kube-apiserver.
 #serviceCIDR: 10.96.0.0/12
+
+# Determines how traffic is encapsulated. It has the following options
+# encap(default): Inter-node Pod traffic is always encapsulated and Pod to outbound traffic is masqueraded.
+# noEncapSameL2:  Inter-node Pod traffic is not encapsulated, but Pod to outbound traffic is masqueraded.
+#                 Worker nodes must be on the same L2 segment.
```

Comment:
Nit: nodes -> Nodes. It is minor, but in most code/documentation, we use Node but not worker Node to follow K8s API, so maybe just use Node.

Reply:
@jianjuns 

I think u r correct all alone. It just dawns on me that NoEncapSameL2 is just subnet of NoEncap(routed) because even if there are multiple subnets on the same l2 segment, there will be a gw interface somewhere for each of subnet.

Will make a quick change
1. move NoEncapSameL2 to NoEncap
2. in noEncap mode, for peer node on different subnet, forward to node gw.

Thx, Su

Reply:
move back to noEncap node, the noEncap mode routing behavior now is
1. if peer node is on the same subnet, peer node is the next hop of pod CIDR
2. if not, node's route table is used to route pod CIDR assuming there is underlying route support.